tests/test-extensions/marvel-extension/src/main/java/org/mule/test/marvel/drstrange/DrStrangeOperations.java

```diff
@@ -67,6 +74,93 @@ public String readStream(@Connection MysticConnection connection, @Optional(defa
     }
   }
 
+  public PagingProvider<MysticConnection, Relic> getRelics(StreamingHelper streamingHelper) {
+    return new PagingProvider<MysticConnection, Relic>() {
+
+      private int currentPage = 1;
+      private final int PAGES = 4;
+
+      @Override
+      public List<Relic> getPage(MysticConnection connection) {
+        if (currentPage == PAGES) {
+          return emptyList();
+        }
+        currentPage++;
+        return new ArrayList<Relic>() {
+
+          {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move the contents of this initializer to a standard constructor or to field initializers. ([squid:S1171](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1171))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7751&issues=AWm78GvHotIXnmsUmYjN&open=AWm78GvHotIXnmsUmYjN)

[ISSUE_KEY]: AWm78GvHotIXnmsUmYjN